packages/invoices/src/InboundShipment/ListView/ListView.tsx

```diff
@@ -51,10 +50,7 @@ export const InboundListView: FC = () => {
 
   const columns = useColumns<InvoiceRow>(
     [
-      // getNameAndColorColumn((row: InvoiceRow, color: Color) => {
-      //   onUpdate({ ...row, color: color.hex });
-      // }),
-      'otherPartyName',
+      [getNameAndColorColumn(), { setter: onUpdate }],
```

Comment:
the colour update in the `ListView` is throwing an error, as its updating the status field too. The version in the detail view / SitePanel is fine; that one is using `useInboundFields` which isn't sending status.

Reply:
Oops! I've changed this mutation to only update colors